Remove Piccadilly Testnet - Piccadilly decommissioning scheduled for 2025-09-26

jay-cousins opened the pull request:
PR to make edits for the Piccadilly Testnet decommissioning, which is scheduled for 2025-09-26.

Full list of edits in #276 

closes #253 and closes #276

jay-cousins committed:
Edits for issues #253 and #276

aiman reviewed (changes requested):
Still references to Piccadilly remaining in several documents

jay-cousins committed:
Remove mention of piccadilly from AGC CLI Reference

jay-cousins commented:
Hi @aiman - I left the Piccadilly mentions in the command-line reference as:

- the flag command-line option is still in the client, the config isn't. Anyone using it in error cannot connect to Piccadilly
- to keep the command-line reference in sync with the AGC command-line output which has the flag.

However, now removed in commit 6bb711e88b80d5e9d18a3f63916d34e0b1424e66

aiman reviewed (changes requested):
If you run a scan of the repo, there are still references to Piccadilly. Please remove them or advise why they are retained.

jay-cousins commented:
> If you run a scan of the repo, there are still references to Piccadilly. Please remove them or advise why they are retained.

```console
% grep -lr "piccadilly" .
./.devenv/state/venv/lib/python3.11/site-packages/eth_utils/__json/eth_networks.json
./developer/deploy-truffle/index.md
./docs/developer/deploy-truffle/index.html
./docs/networks/mainnet/dax.html
./docs/account-holders/setup-nodejsconsole/index.html
./docs/reference/utility-tools/index.html
./networks/mainnet/dax.md
./.git/config
./.git/HEAD
./.git/logs/HEAD
./.git/logs/refs/heads/piccadilly-network
./.git/COMMIT_EDITMSG
./account-holders/setup-nodejsconsole/index.md
./.quarto/xref/6d09eb29
./.quarto/xref/55d4e291
./.quarto/xref/INDEX
./.quarto/idx/developer/deploy-truffle/index.md.json
./.quarto/idx/networks/mainnet/dax.md.json
./.quarto/idx/account-holders/setup-nodejsconsole/index.md.json
./.quarto/idx/reference/utility-tools/index.md.json
./reference/utility-tools/index.md
```
These are all in non-published/live page content, hence no edits.

I will go through them now and cleanse.

jay-cousins committed:
Remove legacy pages and edit WIP pages mentioning Piccadilly

jay-cousins commented:
Hi @aiman cleansed in commit 6d229379bdf5f4aae7850eaaa6563b104fed41c2.

---

I have been through and:

```
./networks/mainnet/dax.md
./docs/networks/mainnet/dax.html
```

Changed to Bakerloo. A WIP draft pending AMM.

```
./developer/deploy-truffle/index.md
./docs/developer/deploy-truffle/index.html
./docs/account-holders/setup-nodejsconsole/index.html
./account-holders/setup-nodejsconsole/index.md
./docs/reference/utility-tools/index.html
./reference/utility-tools/index.md
```
Removed: legacy, hidden pages. Also removed the legacy `submit-trans-nodejsconsole` and `make-calls-nodejsconsole`. Overdue legacy cleanup.


This leaves:

```
% grep -lr "piccadilly" .
./.devenv/state/venv/lib/python3.11/site-packages/eth_utils/__json/eth_networks.json
./.git/config
./.git/HEAD
./.git/logs/HEAD
./.git/logs/refs/heads/piccadilly-network
./.git/COMMIT_EDITMSG
./.quarto/xref/6d09eb29
./.quarto/xref/55d4e291
./.quarto/xref/INDEX
```
The  `python3.11/site-packages/eth_utils` has all the _historical_ listed Bakerloo and Piccadilly networks in it.

The git and quarto are because the branch is named `piccadilly-network` and because of piccadilly appearing in what are now old quarto xref files.

aiman commented:
You can use `git grep` to ignore untracked files: `git grep -i piccadilly`.

aiman reviewed (approved):
LGTM
